docs/io/visualization/using_widgets.rst

```diff
@@ -9,6 +9,82 @@ notebook in action.
 
 Currently, TARDIS supports the following widgets:
 
+Custom Abundance Widget
+#######################
+This widget is a GUI that allows you to develop custom model compositions 
+graphically and output the model to a new file.
+
+.. image:: ../images/custom_abundance_widget.png
+    :alt: Demo of Custom Abundance Widget
+
+The GUI consists of three parts:
+
+1. **Visualization plot** - A step graph generated using `Plotly` that shows 
+abundances and densities as a function of velocity. 
+
+2. **Data editor** - An interface to edit abundances, densities and velocity 
+shells.
+
+3. **File output** - An output module to save the model compositions as a CSVY 
+file.
+
+Interacting with the GUI
+========================
+
+This GUI is developed and composited by `ipywidgets`. Thus you can interact 
+with it in the notebook. There is some main functionality of it.
+
+Edit Abundances
+---------------
+There are two radio buttons which allow you to edit either single shell or 
+mutiple shells. The new input will be applied to selected shell(s) immediately 
+and the plot is updated at the same time. If you want to edit multiple shells 
+at a time, remember to choose the second radio button and set the range of 
+shell number using int slider. The selected shell(s) is highlighted in the 
+plot.
+
+.. image:: ../images/cus_abund_edit_abundance.gif
```

Comment:
This image doesn't show up in docs preview - please fix it!

Reply:
I built the docs again and this issue was fixed then.